<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsyncAnimatorPanel</class>
 <widget class="QDockWidget" name="AsyncAnimatorPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>379</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>379</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>379</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AsyncAnimator</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="nextSkelKeyframe">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addSkelKeyframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSkelAnimation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Skel Anim.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadSkelAnimation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Skel Anim.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="skelKeyframeList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>135</width>
      <height>103</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editSkelTime">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteSkelKeyframe">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="skelLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Skel Keyframes</string>
    </property>
   </widget>
   <widget class="QLabel" name="cageLabel">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cage Keyframes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCageKeyframe">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addCageKeyframe">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="editCageTime">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="cageKeyframeList">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>90</y>
      <width>135</width>
      <height>103</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextCageKeyframe">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCageAnimation">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>40</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Cage Anim.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCageAnimation">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>10</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Cage Anim.</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>10</y>
      <width>16</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bake">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>135</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bake Anim. OBJs!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bakePic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>135</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bake Anim. Pics!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bakePicRest">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>285</y>
      <width>135</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bake An. Rest Pics!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bakeRest">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>260</y>
      <width>135</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bake An. Rest OBJs!</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>245</y>
      <width>300</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>335</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>69</green>
          <blue>70</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>69</green>
          <blue>70</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>84</green>
          <blue>92</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="skelLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Animation Speed</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>315</y>
      <width>300</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
